unicode-data/Changelog.md

```diff
@@ -4,10 +4,10 @@
 
 - Added full case conversions to `Unicode.Char.Case`:
 
-  - Case folding: `caseFoldMapping`, `toCaseFoldString`.
-  - Lower case: `lowerCaseMapping`, `toLowerString`.
-  - Upper case: `upperCaseMapping`, `toUpperString`.
-  - Title case: `titleCaseMapping`, `toTitleString`.
+  - Case folding: `caseFoldMapping`, `caseFoldMapping'`, `toCaseFoldString`.
+  - Lower case: `lowerCaseMapping`, `lowerCaseMapping'`, `toLowerString`.
+  - Upper case: `upperCaseMapping`, `upperCaseMapping'`, `toUpperString`.
+  - Title case: `titleCaseMapping`, `titleCaseMapping'`, `toTitleString`.
```

Comment:
We can remove the alternative versions now.

Reply:
Fixed.